convert-to-pdf.js

```diff
@@ -0,0 +1,34 @@
+const { exec} = require('child_process');
+const waitOn = require('wait-on')
+var opts = {
+    resources: ["http://localhost:3000/"]
+}
+const PORT = "3000";
+
+exec("yarn serve", (error, stdout, stderr) => {
```

Comment:
Should we also do a `build` before this, since `serve` depends on it?

Reply:
Instead of all this manual work to do the child process, let's use https://www.npmjs.com/package/execa

Reply:
> Should we also do a `build` before this, since `serve` depends on it?

In `package.json`, I have ` "serve-convert": "yarn build && node convert-to-pdf.js"`, I do it this way to get better output when it builds.

Reply:
> Instead of all this manual work to do the child process, let's use https://www.npmjs.com/package/execa

Sure, let me have a look